src/data/int/modeq.lean

```diff
@@ -25,9 +25,14 @@ namespace int
 /-- `a ≡ b [ZMOD n]` when `a % n = b % n`. -/
 @[derive decidable]
 def modeq (n a b : ℤ) := a % n = b % n
-
 notation a ` ≡ `:50 b ` [ZMOD `:50 n `]`:0 := modeq n a b
 
+/-- `a ≢ b [ZMOD n]` when `a % n ≠ b % n`. -/
+@[reducible]
+def modneq (n a b : ℤ) := ¬ (a ≡ b [ZMOD n])
+notation a ` ≢ `:50 b ` [ZMOD `:50 n `]`:0 := modneq n a b
+lemma modneq_iff (n a b : ℤ):  a ≢ b [ZMOD n] ↔ a % n ≠ b % n := by refl
```

Comment:
I'd prefer to see these lines separated by newlines.
```suggestion
def modneq (n a b : ℤ) := ¬ (a ≡ b [ZMOD n])

notation a ` ≢ `:50 b ` [ZMOD `:50 n `]`:0 := modneq n a b

lemma modneq_iff (n a b : ℤ) : a ≢ b [ZMOD n] ↔ a % n ≠ b % n := by refl
```
Note that in the lemma I've inserted a space before the colon and deleted a space that came after.

Reply:
Thanks. https://github.com/leanprover-community/mathlib/pull/14682/commits/8c3bdf8899c83650c9ef528859a5eb11e29d0d25